webui/react/src/pages/TrialDetails.tsx

```diff
@@ -232,7 +232,7 @@ const TrialDetailsComp: React.FC = () => {
           <TrialDetailsOverview experiment={experiment} trial={trial} />
         </TabPane>
         <TabPane key={TabType.Hyperparameters} tab="Hyperparameters">
-          <TrialDetailsHyperparameters experiment={experiment} trial={trial} />
+          <TrialRangeHyperparameters experiment={experiment} trial={trial} />
```

Comment:
If it's easy to do we could potentially try to check exp config to understand if exp is single trial or not (maybe by moving the check i added on the exp page in a common util fn) and serve this component only if the exp is not single-trial.

User can still access the trial page via URL, I don't think this will happen, so I would do this small change only if it takes a couple of minutes.